package.json

```diff
@@ -2,7 +2,7 @@
   "name": "the-trans-dimension",
   "scripts": {
     "postinstall": "elm-tooling install",
-    "start": "elm-pages dev --port 3000",
+    "start": "elm-pages dev --port 3030",
```

Comment:
This is fine by me - but also not part of this feature. So I want to double check is deliberate.

Reply:
I moved the port because I run PC locally on 3000 and point TD at localhost and PC is the more complicated project